src/cart/src/utest/test_gurt.c

```diff
@@ -1897,6 +1897,62 @@ test_gurt_atomic(void **state)
 	assert(mix  == 123456);
 }
 
+static void
+check_string_buffer(d_string_buffer_t *str_buf, int str_size, int buf_size,
+	const char *test_str)
+{
+	assert_non_null(str_buf->str);
+	assert_int_equal(str_buf->str_size ,str_size);
```

Comment:
(style)  space prohibited before that ',' (ctx:WxV)
(style)  space required after that ',' (ctx:WxV)